devicetypes/smartthings/zigbee-accessory-dimmer.src/zigbee-accessory-dimmer.groovy

```diff
@@ -122,3 +123,8 @@ def installed() {
 	sendEvent(name: "level", value: 0, isStateChange: false, displayed: false)
 	sendEvent(name: "button", value: "pressed", isStateChange: false, displayed: false)
 }
+
+def configure() {
+    // strangely, these are necessary to have the device report when its buttons are pressed
+	zigbee.onOffConfig() + zigbee.levelConfig() + zigbee.configureReporting(0x0005, 0x0001, DataType.UINT8, 1, 3600, null)
```

Comment:
Guessing you could call `zigbee. addBinding(0x0006)`, etc.... Since this device doesn't have reportable attributes it's only the binding that should matter.

Reply:
yep, that was it